common/lib/xmodule/xmodule/css/annotatable/display.scss

```diff
@@ -85,7 +86,7 @@ $annotatable--body-font-size: em(14);
         &.selected { background-color: $selected_color; }
       }
     }
-    &.highlight-#{$marker} {
```

Comment:
NOTE: fixing some really poorly architected Sass maps in the Harvard Annotations tool styling (which caused an error in the most current version of Sass).